new/LIT7071Qene.xml

```diff
@@ -0,0 +1,73 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7071Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ለሰብአ፡ ማኅበር፡ ጸሪቀ፡ ማኅበሮሙ፡ ጳውሎስ፡ ድኅረ፡ ተቀስመ፡ በፄው፡ ወድኅረ፡ በእሳት፡ በሰለ።</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">La-sabʾa māḫbar ṣariqa māḫbaromu Ṗāwlos dǝḫra taqasma baḍew 
+                    wa-dǝḫra ba-ʾǝsāt basala. (Qǝne of the śǝllāse-type)</title>
+                <title xml:lang="en" corresp="#t1">For the people of the monastery / part of their monastery / Paul / after it was made tasty in salt, 
```

Comment:
It seems to be connected to the martyrdoms of Peter and Paul. The first part, which must be a "synonym" for Paul, is what confuses me. Maybe:

"The communal coin of the people of the community, Paul, after being seasoned with salt and cooked on fire…"